flink-metrics/flink-metrics-influxdb/src/test/java/org/apache/flink/metrics/influxdb/InfluxdbReporterTest.java

```diff
@@ -57,6 +57,7 @@
 	private static final String TEST_INFLUXDB_DB = "test-42";
 	private static final String METRIC_HOSTNAME = "task-mgr-1";
 	private static final String METRIC_TM_ID = "tm-id-123";
+	private String retentionPolicy = "";
```

Comment:
How about using final variable `RETENTION_POLICY` just like above tests?

Reply:
I did use final variable for retention policy in the test case so that the createMetricsRegistry can return the registry impl with  properties both default and non-default test  scenrios.   Please let me now if you have any additiona advice.  Thanks

Reply:
Just personal opinion: I think the second test (`testMetricReportingWithRetentionPolicy`, as it is now) is redundant, and doesn't show some new aspect under test. Alternatives could be, to keep one test and either:
 * make retention policy parameter as `private static final` (with some random, non-empty string);
 * OR, make the `retentionPolicy` variable local in the test method and explicitly pass it to `createMetricRegistry()`.

Reply:
I've added the retention policy argument to createMetricRegistry method and updated tests.